<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterseccionesPluginDialogBase</class>
 <widget class="QDialog" name="InterseccionesPluginDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InterseccionesOSM</string>
  </property>
  <widget class="QPushButton" name="pushButtonGenerar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Generar Shapefile</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEstado">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRuta">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSeleccionarRuta">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar carpeta</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLocalidad">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxProvincia">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar provincial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar localidad</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelZonaUTM">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;UTM Zone: —&quot;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRutaCSV">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBuscarCSV">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar CSV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPesoAtropello">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPesoColision">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPesoVuelco">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x Atropello +</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x Colisión +</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x Vuelco</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>275</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesos separados para cada tipo de siniestro</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
